examples/hello-world/README.md

```diff
@@ -0,0 +1,235 @@
+
+How can I develop a capability
+==============================
+The goal of this section is to show how a developer can develop an OSGI bundle containing a capability, to be deployed in MQNaaS.
+
+Prerequisites
+-------------
+These are the requirements to develop a capability:
+
+* OSGi bundle
+
+  In order to pack and deploy a capability, it is necessary to create an OSGi bundle as a JAR file. It is possible to find more information on OSGi [here](http://www.osgi.org/Technology/WhatIsOSGi). Bundles are the OSGi components made by the developers.
+  
+  In this case the option chosen to build bundles is to use a Maven plugin called [Apache Felix Maven Bundle Plugin](http://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html). It facilitates the job of building the JAR file with the OSGI bundle requisites (basically the MANIFEST headers). This code portion of the project's `pom.xml` file shows the usage of the plugin:
+  
+  ```xml
+  <project>
+    ...
+    <build>
+  		<plugins>
+  		...
+  			<plugin>
+  				<groupId>org.apache.felix</groupId>
+  				<artifactId>maven-bundle-plugin</artifactId>
+  				<extensions>true</extensions>
+  				<configuration>
+  					<instructions>
+  						<Import-Package>*</Import-Package>
+  						<Export-Package>
+  							org.mqnaas.examples.helloworld
+  						</Export-Package>
+  					</instructions>
+  				</configuration>
+  			</plugin>
+  		</plugins>
+  	</build>
+  </project>
+  ```
+  
+  In this case, only `Import-Package` and `Export-Package` instructions are configured. This is a common way to proceed. There are more instructions defined in [Maven parent project POM file](/examples/pom.xml#L127).
+  
+  Moreover, it is necessary to define a different `packaging` for the Maven project. Instead of common `jar` type or similar, `bundle` type must be used. This simple line in the `pom.xml` file defines it:
+  
+  ```xml
+  <project>
+    ...
+    <packaging>bundle</packaging>
+    ...
+  </project>
+  ```
+  
+The capability
+--------------
+### Capability interface
+It is time to develop a capability. First of all, a capability must be defined as a Java interface. Each method of the capability interface will be a service. Any capability Java interface must extend [ICapability Java interface](/core.api/src/main/java/org/mqnaas/core/api/ICapability.java).
+
+In this example capability, there will be only a method receiving a plain Java String and returning another one. This portion of code shows it:
+
+```java
+public interface IHelloWorldCapability extends ICapability {
+  ...
+  public String hello(String name);
+  
+}
+```
+
+This is one of the simplest capability definitions one can develop.
+
+### Capability implementation
+A capability implementation is simply a Java class implementing an ICapability interface. There can be one or more implementations, but only one at the time can be used. In this example, one single implementation is provided and used.
```

Comment:
I would prefer to say "... but only one at the time can be used in a specific Resource", or something like this, but making reference to the resource.